arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1628,16 +1612,16 @@
         },
         "startTimeUtc": {
           "format": "date-time",
```

Comment:
@rkmanda I am testing the `ExportDevices` API recently (with the official generated Python SDK). And I find out that deserialization error will be thrown due to the incorrect format for datetime in the response.
In generated Python SDK, `date-time` format in Swagger is translated to ISO 8601 datetime string. You can find the generated code at [here](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-iothub/azure/mgmt/iothub/models/job_response.py#L46).
I guess this is some inconsistency between swagger definition and server-side implementation. Could you help take a look at this issue? Thanks!

Sample job response:
```json
        {
            "jobId": "23994239-f2cd-4492-912e-2911e7eb0505",
            "startTimeUtc": "Thu, 24 Nov 2016 02:32:11 GMT",
            "endTimeUtc": "Thu, 24 Nov 2016 02:32:46 GMT",
            "type": "export",
            "status": "completed",
            "progress": 100,
            "outputBlobContainerUri": "",
            "excludeKeysInExport": true,
            "useSecondaryStorageAsSource ": false
        }
```
Error message:
```bash
Cannot deserialize datetime object., ValueError: Invalid datetime string: THU, 24 NOV 2016 02:32:11 GMT
```